go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/kubernetes-csi/external-provisioner
 
-go 1.12
+go 1.13
 
 require (
-	github.com/container-storage-interface/spec v1.1.0
-	github.com/golang/mock v1.2.0
+	github.com/container-storage-interface/spec v1.2.0
+	github.com/golang/mock v1.4.3
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
 	github.com/kubernetes-csi/csi-test v2.0.0+incompatible
```

Comment:
Can we update csi-test and external-snapshotter to the latest released versions as well? I'm actually surprised snapshotter is still on 1.2.1? Didn't we need the new beta apis? cc @xing-yang

Reply:
@msau42 Updated csi test and also external snapshotter. As you noticed external-snpahotter was on 1.2.x :( .